borg_import/main.py

```diff
@@ -6,13 +6,18 @@
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--rsnapshot-root", help="Path to saved rsnapshots")
+    parser.add_argument('rsnapshot_root', help='Path to rsnapshots directory')
+    parser.add_argument('-d', '--disk-usage', help='Display rsnapshots disk usage', action="store_true")
     args = parser.parse_args()
 
     if args.rsnapshot_root:
         rsnapshot_root = args.rsnapshot_root
-        for rsnapshot in rsnapshots.get_snapshots(rsnapshot_root):
-            print(rsnapshot)
+        if args.disk_usage:
+            for rsnapshot in rsnapshots.get_disk_usage(rsnapshot_root):
+                print(rsnapshot)
+        else:
```

Comment:
that "else" seems a bit strange, rather directly check for the option triggering this functionality?